Sources/MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -343,7 +343,8 @@ extension ElectronicHorizon {
         public static let positionKey: NotificationUserInfoKey = .init(rawValue: "position")
         
         /**
-         A key in the user info dictionary of a `Notification.Name.electronicHorizonDidUpdatePosition` notification. The corresponding value is an `ElectronicHorizon` at the root of a tree of edges in the routing graph. */
+         A key in the user info dictionary of a `Notification.Name.electronicHorizonDidUpdatePosition` notification. The corresponding value is an `ElectronicHorizonEdge` at the root of a tree of edges in the routing graph. This graph represents a probable path (or paths) of a vehicle within the routing graph for a certain distance in front of the vehicle, thus extending the user’s perspective beyond the “visible” horizon as the vehicle’s position and trajectory change.
```

Comment:
👍 It’s good that we’re putting some more detail here, now that there isn’t a general “electronic horizon” type to look up.